docs/docs/platform/activity-feed.md

```diff
@@ -13,12 +13,14 @@ This can be used to monitor activity and discover potential issues with a specif
 
 To use a subscriber activity feed you can filter the feed by using the subscriberId or email. This will show only the relevant data for this particular subscriber.
 
-You can hover over the status icon to get additional information in case there was an error during delivery.
+You can hover over the status icon to get additional information, in case if there was an error during delivery.
```

Comment:
```suggestion
You can hover over the status icon to get additional information, in case there was an error during delivery.
```

The if is not right here